app.py

```diff
@@ -1,27 +1,121 @@
+from crypt import methods
+from curses.ascii import EM
+import email
 import os
 import json
 from flask import (
     Flask, flash, render_template,
-    redirect, request, session, url_for)
+    redirect, request, session, url_for, jsonify)
+import json
+from flask_login import current_user
 from flask_pymongo import PyMongo
 from bson.objectid import ObjectId
+from werkzeug.security import generate_password_hash, check_password_hash
+# Import wtforms
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, BooleanField
+from wtforms.validators import Email, InputRequired, Length
+# Import authentication forms
+from forms import LoginForm, RegisterForm, ResetPasswordForm
+
+# Import Flask-Bootstrap
+from flask_bootstrap import Bootstrap
```

Comment:
This does not seem to be in the `requirements.txt`. Also, I think that uses Bootstrap 3, right? I think we said we want to use Bootstrap 5.

Reply:
Flask-Bootstrap and bootstrap-flask were most certainly in my requirements.txt file when I originally opened this PR. Looks like it got written over in a merge. I also delete Flask-SQLAlchemy and a bunch of other dependencies we're no longer using and froze it and they are all back now, so I'll have to fix that. We need Bootstrap-Flask, which supports Bootstrap versions 4 & 5 - will fix that too